drivers/hmis/app/graphql/types/hmis_schema/has_external_form_submissions.rb

```diff
@@ -23,7 +23,7 @@ def external_form_submissions_field(name = :external_form_submissions, descripti
       def resolve_external_form_submissions(scope = object.external_form_submissions, filters: nil)
         return [] unless current_user.can_manage_external_form_submissions?
 
-        scope = scope.apply_filters(filters) if filters.present?
+        scope = scope.apply_filters(filters)
```

Comment:
Won't this error if no filters are passed?

Reply:
Confirmed that the api errors when no filters are passed, even though it's an optional arg. It would probably be nice to handle this in the `BaseFilter`, or just adding back this check.